rosidl_tutorials/rosidl_tutorials_msgs/package.xml

```diff
@@ -10,8 +10,8 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
-
   <exec_depend>rosidl_default_runtime</exec_depend>
+  <member_of_group>rosidl_interface_packages</member_of_group>
```

Comment:
I would suggest to add an empty before this one to make the `member_of_group` stand out more.

Reply:
Done